File: net/DomainReputationApi/DomainReputationApiSample.cs
using System;
using System.Net;

using Newtonsoft.Json;

namespace DomainReputationApi
{
    internal static class DomainReputationApiSample
    {
        private static void Main()
        {
            var sample = new DomainReputationApiClient
            {
                ApiKey = "Your domain reputation api key"
            };

            const string domain = "threatintelligenceplatform.com";

            // Download JSON
            var result = sample.SendGet(domain);

            // Print a nice informative string
            PrintResponse(result);
        }

        private static void PrintResponse(string response)
        {
            dynamic responseObject = JsonConvert.DeserializeObject(response);

            if (responseObject != null)
            {
                Console.Write("Score: " + responseObject.reputationScore);
                Console.WriteLine("\n--------------------------------");
                return;
            }

            Console.WriteLine();
        }
    }

    public class DomainReputationApiClient
    {
        public string ApiKey { private get; set; }

        private const string Url =
            "https://api.threatintelligenceplatform.com/v1/reputation";

        public string SendGet(string domain)
        {
            var requestParams = "?domainName=" + Uri.EscapeDataString(domain)
                              + "&apiKey=" + Uri.EscapeDataString(ApiKey);

            var fullUrl = Url + requestParams;

            Console.Write("Sending request to: " + fullUrl + "\n");

            // Download JSON into a string
            var result = new WebClient().DownloadString(fullUrl);

            return result;
        }
    }
}

File: node/domain-reputation-api-sample.js
let https = require('https');
let querystring = require('querystring');

const apiUrl = 'https://api.threatintelligenceplatform.com/v1/reputation';

const parameters = {
    apiKey: 'Your Domain Reputation API key',
    domainName: 'threatintelligenceplatform.com',
    mode: 'full'
};

let url = apiUrl + '?' + querystring.stringify(parameters);

https.get(url, function (res) {
    let rawData = '';

    res.on('data', function(chunk) {
        rawData += chunk;
    });

    res.on('end', function () {
        try {
            let parsedData = JSON.parse(rawData);

            if (parsedData.reputationScore)
                console.log(
                    'Score: ' + parsedData.reputationScore);
            else
                console.dir(parsedData, {colors: true});
        }
        catch (e) {
            console.log(e.message);
        }
    });
}).on('error', function(e) {
    console.log('Error: ' + e.message);
});